roomControl.cs

```diff
@@ -14,18 +14,30 @@
 
 namespace hotel
 {
-    public partial class roomControl : UserControl
+    public partial class RoomControl : UserControl
```

Comment:
Bunun yukaridaki RoomControl.cs dosyasindan farki nedir?

Reply:
daha önce mantıklı bir isimlendirme yapmamışım gibi geldi. bu yüzden isimini değiştirmiştim.

Reply:
Ha dosyanin da adini degistirdin sanirim. Degisen dosyalarda iki tane farkli RoomControl.cs gozukuyor da. Github'in gosterimi ile ilgili bir sorun sanirim.

Reply:
bilmiyorum, belki olabilir çünkü stage tarafından da geçirmiyor bazen. Commit attıktan sonra stage de bırakıyor tekrar eklemem filan gerekiyor.